build/build.cmd

```diff
@@ -0,0 +1,12 @@
+IF "%buildConfiguration%"=="" (SET configuration=Debug) ELSE (SET configuration=%buildConfiguration%)
+
+nuget restore Datadog.Trace.sln
```

Comment:
Unfortunately, Windows batch files don't stop on the first error, which as a dev running this on my box we want. You have to explicitly check for error after each command, IIRC: `if %errorlevel% neq 0 exit /b %errorlevel%`.

Reply:
Suggestion: add a `REM` to the top and mention that on dev boxes this needs a few tools in the path: nuget, msbuild, dotnet. And that these tools are typically available on Visual Studio command prompt

Reply:
I can confirm that if nuget is installed and running the script from the Developer Command prompt it works @dszmigielski 👍

Reply:
@pjanotti not sure if it will not be easier to rewrite it to a bash script which you can use on Windows e.g. from Git Bash. thanks to it I imagine the script could work on MacOS and Linux if mono is installed... does it make any sense?

Reply:
Yes, @pellared I like the idea - it may be easier and more portable. My question is: would we be able to run the same bash script on the Windows build on GH action? We started with the idea of using the build steps in a script so we could leverage it locally so if it is possible to run it on GH action on Windows then it is fine for me.